include/small/RouteLayer.hpp

```diff
@@ -0,0 +1,72 @@
+//****************************************************************************
+// SMaLL, Software for Machine Learning Libraries
+// Copyright 2023 by The SMaLL Contributors, All Rights Reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// For additional details (including references to third party source code and
+// other files) see the LICENSE file or contact permission@sei.cmu.edu. See
+// Contributors.txt for a full list of contributors. Created, in part, with
+// funding and support from the U.S. Government (see Acknowledgments.txt file).
+// DM23-0126
+//****************************************************************************
+
+#pragma once
+
+#include <small.h>
+#include <small/buffers.hpp>
+#include <small/Layer.hpp>
+
+namespace small
+{
+
+//****************************************************************************
+template <typename BufferT>
+class RouteLayer : public Layer<BufferT>
+{
+public:
+    typedef typename BufferT::value_type value_type;
+
+    RouteLayer(shape_type const &input1_shape,
+             shape_type const &input2_shape)
+        : Layer<BufferT>()
+    {
+#if defined(DEBUG_LAYERS)
+        std::cerr << "Add(batches:" << input1_shape[BATCH]
```

Comment:
Add --> Route

Reply:
Fixed in [f419425](https://github.com/CMU-SPEED/SMaLLFramework/pull/7/commits/f4194253ec542140623dcae1d5440638c1b3ee99)